src/Microsoft.DotNet.Build.CloudTestTasks/RunnerScripts/scriptrunner/scriptrunner.py

```diff
@@ -81,8 +83,9 @@ def _main(settings, optlist, args):
         else:
             log.error("Error: No exception thrown, but XUnit results not created")
             test_executor.report_error(settings, failure_type="XUnitTestFailure")
+            return WORKITEM_FAILED
```

Comment:
Well that seems like our bug in that case. We shouldn't be hiding important information from the users.